addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -36,19 +33,19 @@ export class ChannelMember extends Record {
         const [command, memberData] = Array.isArray(data) ? data : ["ADD", data];
         this.id = memberData.id;
         if ("persona" in memberData) {
-            this.persona = this._store.Persona.insert({
+            this.persona = {
                 ...(memberData.persona.partner ?? memberData.persona.guest),
                 type: memberData.persona.guest ? "guest" : "partner",
                 country: memberData.persona.partner?.country,
                 channelId: memberData.persona.guest ? memberData.channel.id : null,
-            });
+            };
         }
         let thread = memberData.thread ?? this.thread;
         if (!thread && memberData.channel?.id) {
-            thread = this._store.Thread.insert({
+            thread = {
```

Comment:
note: very confusing variable here, with mix of type

Reply:
It's a bit confusing, but `thread` could actually be a thread above.
Would be cumbersome to define another variable for the data part.
All in all, this code will not live for long, with the incoming simplification of insert/update in followup PR